assets/_fonts.scss

```diff
@@ -1,12 +1,41 @@
 $font-path: "/fonts";
 
+/*!
```

Comment:
@marina-chibizova While I'm adding a bunch of lines here they are actually carried over from the theme respective [file](https://github.com/alex-shpak/hugo-book/blob/d86d5e70c7c0d787675b13d9aee749c1a8b34776/assets/_fonts.scss).

Reply:
makes sense. I would change the collapsible icon though, to just an arrow without the circle

Reply:
> to just an arrow without the circle

ok